File: hot-cli/src/main/resources/examples/example.show.py
s = "Hello My friend"

def printHello(request):
    return s
    
def printJson(request):
	return {"items":{"name":"damien", "age":8}}
	
def onconnect(connection):
	
	def ondata():
		connection.write(data)
	
	def onclose():
		print 'close'
		
	connection.data(ondata)
	connection.close(onclose)

show.rest.get("/pyitems").headers(["Accept: text/plain"]).then(printHello)

show.rest.get("/pyitems").headers(["Accept: application/json"]).then(printJson)

show.rest.put("/pyitem").then(printHello)

show.websocket.addHandler({'path':'/pychatroom'}).connect(onconnect)

File: hot-cli/src/main/resources/examples/example.show.js
var hello = "var get method"

show.rest.get("/jsitems").headers(["Accept: text/plain"]).then(function() {
	return hello
})

show.rest.get("/jsitems").headers(["Accept: application/json"]).then(function(request) {
	return {items:{name:"damien", age:8}}
})

show.rest.put("/jsitem").then(function (){
	return "put method"
})

show.websocket.addHandler({path:"/jschatroom"}).connect(function(connection) {
	connection.data(function(data) {
		connection.write(data)
	})
	connection.close(function(){ hprint("closed") })
})